MetriCam2_SDK.sln

```diff
@@ -94,116 +94,210 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sick.VisionaryT.Pro", "Beta
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AzureKinect", "BetaCameras\AzureKinect\AzureKinect.csproj", "{9A292249-66EE-4C9C-A6B7-DD5112E3625F}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Hikvision", "BetaCameras\Hikvision\Hikvision.csproj", "{8B343646-35D9-4697-A466-14161ECB48CE}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|x64 = Debug|x64
+		Release|Any CPU = Release|Any CPU
 		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{342F049E-668B-4351-B21D-75E4BEF5A1E4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Please do not reintroduce `AnyCPU` build config for whole solution. Build Hikvision only in `x64`.